src/services/routes/org.routes.ts

```diff
@@ -1,16 +1,28 @@
 import type { RouteOptions } from 'fastify';
-import isValidDomain from 'is-valid-domain';
 
 import { HTTP_STATUS_CODES } from '../http.js';
 import type { PluginDone } from '../types/PluginDone.js';
 import { ORG_SCHEMA } from '../schema/org.schema.js';
 import type { FastifyTypedInstance } from '../fastify.js';
-
-export enum ProblemType {
-  MALFORMED_ORG_NAME = 'https://veraid.net/problems/malformed-org-name',
-  MALFORMED_AWALA_ENDPOINT = 'https://veraid.net/problems/malformed-awala-endpoint',
+import { createOrg } from '../../org.js';
+import { CreationProblemType } from '../../CreationProblemType.js';
+
+const ORG_ROUTES_ERROR_MAPPING: {
+  [key in CreationProblemType]: (typeof HTTP_STATUS_CODES)[keyof typeof HTTP_STATUS_CODES];
+} = {
+  [CreationProblemType.EXISTING_ORG_NAME]: HTTP_STATUS_CODES.CONFLICT,
+  [CreationProblemType.MALFORMED_AWALA_ENDPOINT]: HTTP_STATUS_CODES.BAD_REQUEST,
+  [CreationProblemType.MALFORMED_ORG_NAME]: HTTP_STATUS_CODES.BAD_REQUEST,
+} as const;
+
+interface OrgUrls {
+  self: string;
 }
 
+const formUrls = (name: string): OrgUrls => ({
```

Comment:
I'd say "form" is correct but it could potentially be ambiguous. How about `makeUrls()`?